range_generator.py

```diff
@@ -0,0 +1,14 @@
+def MyRange(start, stop, step):
```

Comment:
What about cases where some arguments are not provided (see Python `range`)?

Reply:
I managed to do this, but only using *args ( to keep arguments' order)

Reply:
it's in the function definition below